web/client/actions/geostory.js

```diff
@@ -82,6 +83,15 @@ export const update = (path, element, mode = "replace") => ({
     mode
 });
 
+/**
+ * updates the current page with current value of sectionId (future can be extended adding other info about current content).
+ * @param {object} param0 current page information. Contains `sectionId`
+ */
+export const updateCurrentPage = ({sectionId}) => ({
```

Comment:
hi, not so crucial, but there is no test for this action creator. Please can you add it?

Reply:
reducer's test checks it for the moment